tests/repo_utils/test_method_diff.py

```
@@ -74,27 +74,24 @@ def methods(self) -> list[MethodEntry]:
     def test_new_function_in_mixed_hunk_is_added(self, mock_parse, methods, file_path, changes, hunks):
         mock_parse.return_value = hunks
 
-        get_method_statuses_for_file(methods, file_path, changes, Path("/fake/repo"))
+        by_name = compute_method_statuses_for_file(methods, file_path, changes, Path("/fake/repo"))
 
-        by_name = {m.qualified_name: m.status for m in methods}
         # merge_clusters is entirely new code – it should be ADDED
```

Ruff (0.15.10)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)